sdk/core/Azure.Core/src/Shared/ResponseExceptionExtensions.cs

```diff
@@ -30,16 +31,18 @@ public static RequestFailedException CreateRequestFailedException(this Response
 
         public static RequestFailedException CreateRequestFailedException(this Response response, string message)
         {
-            return CreateRequestFailedExceptionAsync(message, response, false).EnsureCompleted();
+            ValueTask<RequestFailedException> messageTask = CreateRequestFailedExceptionAsync(message, response, false);
+            Debug.Assert(messageTask.IsCompleted);
```

Comment:
Was this intentional?

Reply:
Yep, `EnsureCompleted` is not a shared source yet.